### PR TITLE
Pass bounds to post-process functions.

### PR DESCRIPTION
**Pull request by zerebubuth**
Pass unpadded bounds to any post-process function with 'bounds' as a kwarg.

This seemed preferable to passing it to all functions and having to change the function signature of all the functions. Although if we find ourselves doing this again, might be better to pass a context object which we can extend.

Connects to mapzen/vector-datasource#502.

@rmarianski could you review, please?

### CODE REVIEW

File: tilequeue/process.py [hunk 1]
```diff
@@ -65,12 +66,18 @@ def _preprocess_data(feature_layers, shape_padded_bounds):
...
     for step in post_process_data:
         fn = loadClassPath(step['fn_name'])
         params = step['params']
 
+        # inspect the function to tell if it has kwarg called 'bounds'
+        # and, if so, update the params to include the tile coordinate.
+        if 'bounds' in getargspec(fn).args:
+            params = params.copy()
+            params['bounds'] = bounds
+
```
Comment (rmarianski) [inline tilequeue/process.py]: We might want to explicitly just pass bounds to every single function, or grow a context/state/env variable that has got it in it and always pass everything along there.

Comment (rmarianski) [inline tilequeue/process.py]: Ah nm, saw that you had the same exact thought in the pr description :)

**Comment by rmarianski** [comment]
:+1:

File: tilequeue/process.py (hunk 1 above)
Comment (zerebubuth) [inline tilequeue/process.py]: Yeah, the question is, do we want to do that _now_ or wait until later... I've added #70 to track it.

**Pull request merged**